src/clr-angular/data/tree-view/tree-node.spec.ts

```diff
@@ -186,6 +209,20 @@ export default function(): void {
         this.clarityDirective.expanded = false;
         expect(this.hostComponent.expanded).toBeFalse();
       });
+
+      it(
+        'offers a [(clrDisabled)] two-way binding',
+        fakeAsync(function(this: Context) {
```

Comment:
This is sync, don't need to wrap in `fakeAsync`. See the EventEmitter which is set to sync.